client/app.jsx

```diff
@@ -26,10 +26,13 @@ class App extends React.Component {
   }
 
   componentDidMount() {
+    var urlParams = new URLSearchParams(window.location.search);
```

Comment:
A suggestion.. create a file in your proxy that uses this same code to check if the url id exists, if it's null (no id was passed to the url), then generate a random number 1-100 that will be set to window.id to be used as the id for the whole app - all services. if the id WAS passed to url, then set that to window.id. The code on line 35 looks good for this servce.. the case for which this random number would be triggered, is if you are rendering your service by itself, you wouldn't have the proxy to check for or generate the id number